<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>construction</class>
 <widget class="QWidget" name="construction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>5</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="playerView">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="flagView">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="randomCommonTankView">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="commonTankView">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="randomArmoredTankView">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>210</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="armoredTankView">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>210</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="brickView">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>300</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="waterView">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>300</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="stoneView">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>390</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="bonusItemsBox">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>470</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>BONUSES</string>
   </property>
   <widget class="QCheckBox" name="tankiCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanki</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>clock</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="starCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Star</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>600</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.102273 rgba(0, 0, 0, 255), stop:0.755682 rgba(58, 0, 215, 255));</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>600</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.102273 rgba(0, 0, 0, 255), stop:0.755682 rgba(0, 215, 58, 255));</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>500</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>530</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>565</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="crossView">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>390</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:transparent;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
